### ISSUE TITLE
[TypeScript] Runtime error in IE 11

### ISSUE THREAD

**Issue by tonkotsuboy** [parcel-bundler/parcel#205]
🐛 bug report

### 🎛 Configuration (.babelrc, package.json, cli command)

`tsconfig.json :`

```json
{
  "compilerOptions": {
    "module": "commonjs",
    "lib": [
      "dom",
      "es2015.promise",
      "es5"
    ],
    "target": "es5",
    "noImplicitAny": false,
    "sourceMap": true
  }
}
```

`Main.ts`

```js
alert("typescript works!");
```

`index.js`

```js
import "./ts/Main.ts";
```

`index.html`

```html
<!doctype html>
<html>
<head>
  <script src="index.js"></script>
</head>
<body>
</body>
</html>
```

### 🤔 Expected Behavior
works in IE 11

### 😯 Current Behavior
does not works in IE 11

### 💁 Possible Solution

I guess arrow function in compiled code is the reason

```js
module.bundle.Module = Module;

if (!module.bundle.parent) {
  var ws = new WebSocket('ws://localhost:59462/');
  ws.onmessage = (e) => {
    var data = JSON.parse(e.data);

```

https://github.com/parcel-bundler/parcel/blob/0ff76bea36135a4b1edbd85a588a4e26c86dcc19/src/builtins/hmr-runtime.js#L19

### 🌍 Your Environment

| Software         | Version(s)
| ---------------- | ----------
| Parcel           |  1.1.09
| Node             | 9.1.0
| npm/Yarn         | npm 5.5.1
| Operating System |  macOS High Sierra 10.13.1

**Comment by azu** [comment]
It seems to be related with #169

**Comment by davidnagli** [comment]
Closed by #169 (merged)
